deepset_cloud_sdk/cli.py

```diff
@@ -66,17 +78,19 @@ def list_files(
     batch_size: int = 10,
     timeout_s: int = 300,
 ) -> None:
-    """List files in deepset Cloud.
-
-    A CLI method to list files that exist in deepset Cloud.
+    """List files that exist in the specified deepset Cloud workspace.
 
     :param api_key: deepset Cloud API key to use for authentication.
     :param api_url: API URL to use for authentication.
-    :param workspace_name: Name of the workspace to list the files from.
+    :param workspace_name: Name of the workspace to list the files from. Uses the workspace from the .EVN file by default.
```

Comment:
Our linting is not happy about this: https://github.com/deepset-ai/deepset-cloud-sdk/actions/runs/5344918400/jobs/9689892588?pr=71#step:5:19

The descripts of params are required to be less than 120 chars by default python standards. Technically we can increase it. I will double check whats the best practice here.

Reply:
I'd vote for let's make it as long as needed for it to be clear :)

Reply:
Increased it to 150 chars